packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -278,7 +278,7 @@ export const updateInlineLockKey = 'CODEWHISPERER_INLINE_UPDATE_LOCK_KEY'
 
 export const newCustomizationMessage = 'You have access to new CodeWhisperer customizations.'
 
-export const newCustomizationAvailableKey = 'CODEWHISPERER_NEW_CUSTOMIZATION_AVAILABLE'
+export const newCustomizationsAvailableKey = 'CODEWHISPERER_NEW_CUSTOMIZATIONS_AVAILABLE_AMOUNT'
```

Comment:
since we're renaming the key let's follow the context key naming guidelines https://github.com/aws/aws-toolkit-vscode/blob/master/docs/arch_runtime.md#vscode-context-keys

```suggestion
export const newCustomizationsAvailableKey = 'aws.amazonq.codewhisperer.newCustomizations'
```